client/pages/checkout-form.jsx

```diff
@@ -52,7 +52,7 @@ export default function CheckoutForm(props) {
     const { error } = await stripe.confirmPayment({
       elements,
       confirmParams: {
-        return_url: 'http://localhost:3000/#confirmationPage'
+        return_url: new URL('/#confirmationPage', window.location).href
```

Comment:
This will need to be updated when you set up your back end to handle the order confirmation. 

something like 

```js
return_url: new URL('/process-order', window.location).href
```